plugins/regex_revalidate/regex_revalidate.c

```diff
@@ -251,9 +256,9 @@ prune_config(invalidate_t **i)
     iptr  = *i;
     ilast = NULL;
     while (iptr) {
-      if (difftime(iptr->expiry, now) < 0) {
-        TSDebug(PLUGIN_NAME, "Removing %s expiry: %d type: %s now: %d", iptr->regex_text, (int)iptr->expiry,
-                strForResult(iptr->new_result), (int)now);
+      if (iptr->expiry <= now) {
+        TSDebug(PLUGIN_NAME, "Removing %s expiry: %jd type: %s now: %jd", iptr->regex_text, (intmax_t)iptr->expiry,
+                strForResult(iptr->new_result), (intmax_t)now);
```

Comment:
Why don' t you wanna do these C++ style not C style?  `intmax_t(iptr->expiry)` not `(intmax_t)iptr->expiry`.

Reply:
because the plugin is still 'c' and it won't compile.